applications/DG-Max/Algorithms/DGMaxDiscretization.cpp

```diff
@@ -805,12 +805,20 @@ std::map<typename DGMaxDiscretizationBase::NormType, double>
         for (typename Base::MeshManipulator<DIM>::FaceIterator it =
                  mesh.faceColBegin();
              it != end; ++it) {
+            if (!(*it)->isOwnedByCurrentProcessor()) {
+                continue;
+            }
             dgNorm += faceIntegrator_.integrate(
                 *it, [&](Base::PhysicalFace<DIM>& face) {
                     return faceErrorIntegrand(face, timeVector, electricField);
                 });
         }
     }
+#ifdef HPGEM_USE_MPI
```

Comment:
I see, here you add the reduce, before it was assumed it was one single processor. So each was computing its own error norm and not sharing.

Reply:
Indeed, for reference we are basically computing an integral over all the elements or faces of the mesh, computing the local error on each part.